### ISSUE TITLE
FE | GA Ticket (same endpoint just updating source system, use the new source in the event) (delve into all the questions on refinement)

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#61095]

**Comment by adamwhitlock1** [comment]
There is a widget set up in the sandbox GA dashboard: https://analytics.google.com/analytics/web/#/dashboard/DRqBrmiyTD6l8L75rei0fw/a50123418w177519031p176188361/

It shows that there are several events with EVSS as the source, since the lighthouse api integration is not ready yet in production.

Failures will only show the EVSS source in the event name if the user was authenticated, was a veteran, and the call to get the disability rating failed on the EVSS side. Currently there are lots of events where the call is made, but it is unsuccessful because the user is not a veteran, so it never calls the EVSS service and fails before that.

This should be good to close out now as validated @mtcA6 if you could take a look as well to verify.

![Screenshot 2023-07-18 at 3.25.54 PM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/87c10376-b27b-4c85-b88d-0fc76759ecf3)

We currently dont have an actual dashboard in use that has that event being tracked, so thats why I just made one in the sandbox, soooo yea not sure if it should go somewhere else or what.

**Comment by mtcA6** [comment]
@adamwhitlock1 are there steps we need to take to add this to prod GA?

**Comment by adamwhitlock1** [comment]
No, GA events propagate into prod as soon as they are merged and deployed daily, unless they were explicitly put behind a feature toggle themselves, which I don't think has ever been done/needed. 

The screenshot above is of the current production events.

**Comment by mtcA6** [comment]
@adamwhitlock1 I see when you say sandbox you didn't mean a lower environment, but instead on our 'sandbox' in GA. I think where you put it is good enough for now. 

So the first two rows in that screenshot are LH and the last is EVSS? I renamed the widget that title make sense? 
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/7ce6fe4e-8b2c-45ba-b7f8-b20f7541dd85)


@Samara-Strauss that seems like a significant number of failures. Is that normal?

**Comment by adamwhitlock1** [comment]
So, actually the first event is all the 'old' events where the source was not attached. If you adjust your GA dashboard to just look at the last day, then they will not be present at all, because the events with the source added have effectively replaced that event. 

There are no events that show Lighthouse as the source, because the BE work for that is still ongoing and not launched to production. Once it is launched you will start seeing events with that in the name instead of EVSS

The failures are largely due the fact that any users that are not veterans or do not have a disability rating, the application will still attempt to request the rating info, but the endpoint responds with failure due to them not having access to that endpoint.
